wordpress-mobile/gutenberg-mobile
# Prevent rendering of settings button on Video block

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1071

gutenberg PR: https://github.com/WordPress/gutenberg/pull/16000

To test:

- Test in both iOS & Android
- Verify that Video block doesn't have a settings button

Update release notes:
